### ISSUE TITLE
remote mode issue with multi-users

### ISSUE THREAD

**Issue by JunweiSUN** [microsoft/nni#2688]
**Environment**:
- NNI version: v1.7
- NNI mode (local|remote|pai): remote
- Client OS: ubuntu 16.04
- Server OS (for remote mode only): ubuntu 16.04
- Python version: 3.7
- PyTorch/TensorFlow version: 1.5
- Is conda/virtualenv/venv used?: conda
- Is running in Docker?: no

**Log message**:
 - nnimanager.log: 
[7/15/2020, 10:00:45 AM] INFO [ 'Datastore initialization done' ]
[7/15/2020, 10:00:45 AM] INFO [ 'RestServer start' ]
[7/15/2020, 10:00:45 AM] INFO [ 'Construct remote machine training service.' ]
[7/15/2020, 10:00:45 AM] INFO [ 'RestServer base port is 9092' ]
[7/15/2020, 10:00:45 AM] INFO [ 'Rest server listening on: http://0.0.0.0:9092' ]
[7/15/2020, 10:00:46 AM] INFO [ 'NNIManager setClusterMetadata, key: machine_list, value: [{"ip":"10.112.62.11","username":"nni","passwd":"nni","useActiveGpu":true,"port":22}]' ]
[7/15/2020, 10:00:46 AM] INFO [ 'connecting to nni@10.112.62.11:22' ]
[7/15/2020, 10:00:47 AM] INFO [ 'connected to nni@10.112.62.11:22' ]
[7/15/2020, 10:00:47 AM] INFO [ 'NNIManager setClusterMetadata, key: nni_manager_ip, value: {"nniManagerIp":"10.112.181.126"}' ]
[7/15/2020, 10:00:47 AM] INFO [ 'NNIManager setClusterMetadata, key: trial_config, value: {"command":"/home/sjw/anaconda3/envs/nni/bin/python3 mnist.py","codeDir":"/home/sjw/remote_test/nni/examples/trials/mnist-pytorch/.","gpuNum":1}' ]
[7/15/2020, 10:00:47 AM] INFO [ 'Starting experiment: CFyQ48oA' ]
[7/15/2020, 10:00:47 AM] INFO [ 'Change NNIManager status from: INITIALIZED to: RUNNING' ]
[7/15/2020, 10:00:47 AM] INFO [ 'Add event listeners' ]
[7/15/2020, 10:00:47 AM] INFO [ 'RestServer start' ]
[7/15/2020, 10:00:47 AM] INFO [ 'RestServer base port is 9093' ]
[7/15/2020, 10:00:47 AM] INFO [ 'Run remote machine training service.' ]
[7/15/2020, 10:00:48 AM] INFO [ 'NNIManager received command from dispatcher: ID, ' ]
[7/15/2020, 10:00:48 AM] INFO [ 'NNIManager received command from dispatcher: TR, {"parameter_id": 0, "parameter_source": "algorithm", "parameters": {"batch_size": 32, "hidden_size": 256, "lr": 0.001, "momentum": 0.5778345520006561}, "parameter_index": 0}' ]
[7/15/2020, 10:00:52 AM] INFO [ 'submitTrialJob: form: {"sequenceId":0,"hyperParameters":{"value":"{\\"parameter_id\\": 0, \\"parameter_source\\": \\"algorithm\\", \\"parameters\\": {\\"batch_size\\": 32, \\"hidden_size\\": 256, \\"lr\\": 0.001, \\"momentum\\": 0.5778345520006561}, \\"parameter_index\\": 0}","index":0}}' ]
[7/15/2020, 10:00:53 AM] WARNING [ 'Scheduler: trialJob id H91qb, no machine can be scheduled, return TMP_NO_AVAILABLE_GPU ' ]
[7/15/2020, 10:00:53 AM] INFO [ 'Right now no available GPU can be allocated for trial H91qb, will try to schedule later' ]
[7/15/2020, 10:00:56 AM] WARNING [ 'Scheduler: trialJob id H91qb, no machine can be scheduled, return TMP_NO_AVAILABLE_GPU ' ]
[7/15/2020, 10:00:56 AM] INFO [ 'Right now no available GPU can be allocated for trial H91qb, will try to schedule later' ]
[7/15/2020, 10:00:59 AM] WARNING [ 'Scheduler: trialJob id H91qb, no machine can be scheduled, return TMP_NO_AVAILABLE_GPU ' ]
[7/15/2020, 10:00:59 AM] INFO [ 'Right now no available GPU can be allocated for trial H91qb, will try to schedule later' ]
[7/15/2020, 10:01:02 AM] WARNING [ 'Scheduler: trialJob id H91qb, no machine can be scheduled, return TMP_NO_AVAILABLE_GPU ' ]
[7/15/2020, 10:01:02 AM] INFO [ 'Right now no available GPU can be allocated for trial H91qb, will try to schedule later' ]
[7/15/2020, 10:01:05 AM] WARNING [ 'Scheduler: trialJob id H91qb, no machine can be scheduled, return TMP_NO_AVAILABLE_GPU ' ]
[7/15/2020, 10:01:05 AM] INFO [ 'Right now no available GPU can be allocated for trial H91qb, will try to schedule later' ]
[7/15/2020, 10:01:08 AM] WARNING [ 'Scheduler: trialJob id H91qb, no machine can be scheduled, return TMP_NO_AVAILABLE_GPU ' ]
[7/15/2020, 10:01:08 AM] INFO [ 'Right now no available GPU can be allocated for trial H91qb, will try to schedule later' ]
[7/15/2020, 10:01:11 AM] WARNING [ 'Scheduler: trialJob id H91qb, no machine can be scheduled, return 

**What issue meet, what's expected?**:
When user use remote mode (e.g., user on remote server is user0), nni will create an directory in /tmp/nni to store the experiment information, the owner of this directory will be user0 and the permission of /tmp/nni/experiment is 775:
```
# ls -l /tmp/nni
total 52
drwxrwxrwx   3 user0  user0   4096 Jul 15 02:05 ./
drwxrwxrwt 125 root root 45056 Jul 15 02:12 ../
drwxrwxr-x   3 user0  user0   4096 Jul 15 02:05 experiments/
```
Once this dir is created, if someone want to use another remote account (e.g., user1) to run his experiment, the first trial will be stuck at `waiting` status, to solve this problem, user needs to remove /tmp/nni dir and restart the experiment.

**Suggestion**:
Create an unique directory for each remote user (e.g., /tmp/user0-nni, /tmp/user1-nni) instead of store all the experiments in /tmp/nni

**Comment by JunweiSUN** [comment]
fixed in PR #2689
